packages/block-library/src/embed/embed-preview.native.js

```diff
@@ -36,6 +37,14 @@ const EmbedPreview = ( {
 } ) => {
 	const [ isCaptionSelected, setIsCaptionSelected ] = useState( false );
 
+	const wrapperStyle = styles[ 'embed-preview__wrapper' ];
+	const wrapperAlignStyle =
+		styles[ `embed-preview__wrapper--align-${ align }` ] ||
+		styles[ 'embed-preview__wrapper--align' ];
```

Comment:
I couldn't see a `embed-preview__wrapper--align` object in `styles.native.scss`, should we maybe add it or maybe we can remove this fallback?

Reply:
Good point, originally I defined the style `embed-preview__wrapper--align` but in the last commits, I removed it because it wasn't necessary, so we can safely remove the fallback ✂️ 😄 .

Reply:
This has been addressed in [this commit](a88818615e6db3514a494ed60a33fb902c412589).